client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -93,10 +102,24 @@ define([
                 mainItem: post.groups[1].refs[0],
                 items: post.groups[1].refs
             });
-            // complete Post With User Information
-            userList.getUser(post.original_creator).then(function(user) {
-                post.user = user;
+            post.comments = _.reduce(post.groups[1].refs, function(is, val) {
+                return is || _.isUndefined(val.item.name);
+            }, false);
+            // check if `fullDetails` flag is needed
+            // comments items set falg to true.
+            post.fullDetails = post.comments;
```

Comment:
Ok if this is business logic. Then you probably need to document these fields better and the names should be more explicit (`comments` here suggest a list but this is actually a boolean and the purpose of `fullDetails` is not obvious here).
Also there is a `angular.extends` block just above. I think you should move these new field attributions into it.